projects/igniteui-angular/src/lib/tabs/tabs/tabs.component.html

```diff
@@ -1,5 +1,5 @@
 <div #headerContainer class="igx-tabs__header">
-    <button #scrollPrevButton type="button" igxIconButton="flat" igxRipple class="igx-tabs__header-button" (click)="scrollPrev()">
+    <button #scrollPrevButton type="button" igxIconButton="flat" igxRipple tabindex="-1" aria-hidden="true" class="igx-tabs__header-button" (click)="scrollPrev()">
```

Comment:
This seems to cause a console error on clicking one of the scroll buttons:
![image](https://github.com/user-attachments/assets/b314d1e1-6fed-4cdd-a564-55cd4924741b)

We've recently had a similar issue in the Query Builder https://github.com/IgniteUI/igniteui-angular/issues/14650.